src/components/selects-request/archive-list/table/colums.tsx

```diff
@@ -0,0 +1,80 @@
+import moment from 'moment';
+
+import { P } from '@src/component/atoms';
+import { SelectsRequestStatusToKor } from '@src/types';
+import { getUploaderTypeIconComponent } from '@src/lib/User';
+
+const Time = ({ text }) => {
+  if (text == null) {
+    return null;
+  }
+
+  return <P>{`${moment(text).format('YYYY.MM.DD HH:MM')}`}</P>;
```

Comment:
```suggestion
  return <P>{moment(text).format('YYYY.MM.DD HH:MM')}</P>;
```

변수값을 그대로 쓸땐 template literal로 만들지 않아도 됩니다!

Reply:
eea6961